metal/datasource_metal_device_bgp_neighbors.go

```diff
@@ -9,48 +9,58 @@ func bgpNeighborSchema() *schema.Resource {
 	return &schema.Resource{
 		Schema: map[string]*schema.Schema{
 			"address_family": {
-				Type:     schema.TypeInt,
-				Computed: true,
+				Type:        schema.TypeInt,
+				Description: "IP address version, 4 or 6",
+				Computed:    true,
 			},
 			"customer_as": {
-				Type:     schema.TypeInt,
-				Computed: true,
+				Type:        schema.TypeInt,
+				Description: "Local autonomous system number",
+				Computed:    true,
 			},
 			"customer_ip": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Description: "Local used peer IP address",
+				Computed:    true,
 			},
 			"md5_enabled": {
-				Type:     schema.TypeBool,
-				Computed: true,
+				Type:        schema.TypeBool,
+				Description: "Whether BGP session is password enabled",
+				Computed:    true,
 			},
 			"md5_password": {
-				Type:      schema.TypeString,
-				Computed:  true,
-				Sensitive: true,
+				Type:        schema.TypeString,
+				Description: "BGP session password in plaintext (not a checksum)",
+				Computed:    true,
+				Sensitive:   true,
 			},
 			"multihop": {
-				Type:     schema.TypeBool,
-				Computed: true,
+				Type:        schema.TypeBool,
+				Description: "Whether the neighbor is in EBGP multihop session",
+				Computed:    true,
 			},
 			"peer_as": {
-				Type:     schema.TypeInt,
-				Computed: true,
+				Type:        schema.TypeInt,
+				Description: "Peer AS number (different than customer_as for EBGP)",
+				Computed:    true,
 			},
 			"peer_ips": {
-				Type:     schema.TypeList,
-				Optional: true,
-				Elem:     &schema.Schema{Type: schema.TypeString},
+				Type:        schema.TypeList,
+				Description: "Array of IP addresses of this neighbor's peers",
+				Optional:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
 			},
 			"routes_in": {
-				Type:     schema.TypeList,
-				Computed: true,
-				Elem:     bgpRouteSchema(),
+				Type:        schema.TypeList,
+				Description: "Array of incoming routes. Each route has attributes:",
```

Comment:
was this a multiline description that got cut in the move?

Reply:
yes